stdlib/contextvars.pyi

```diff
@@ -38,11 +38,11 @@ def copy_context() -> Context: ...
 class Context(Mapping[ContextVar[Any], Any]):
```

Comment:
Maybe I'm missing something, but can't we inherit from `Mapping[ContextVar[_T], _T]` and then remove most of the specific methods on this class?

Reply:
I was thinking about doing something similar, but [this comment](https://github.com/python/typeshed/blame/18d746b6b0d96c43c9d4b0221425958e20e212ae/stdlib/contextvars.pyi#L36-L38) persuaded me not to.

If you think that it is time to refactor this, I will be happy to 👍

Reply:
I see, that actually makes sense.